windows/src/desktop/kmshell/install/Keyman.Configuration.System.TIPMaintenance.pas

```diff
@@ -152,7 +156,7 @@ class function TTIPMaintenance.DoInstall(const KeyboardID,
   if RegistrationRequired then
   begin
     // This calls back into TTIPMaintenance.RegisterTip
-    if WaitForElevatedConfiguration(0, '-register-tip '+IntToHex(LangID,4)+' "'+KeyboardID+'" "'+lang.BCP47Code+'"') <> 0 then
+    if WaitForElevatedConfiguration(0, '-register-tip '+IntToHex(LangID,4)+' "'+KeyboardID+'" "'+lang.BCP47Code+'" '+GetUserDefaultLangParameterString) <> 0 then
```

Comment:
I see `-register-tip` and `GetUserDefaultLangParameterString` both use `IntToHex(LangID,4)`, 
where GetUserDefaultLangParameterString gets an updated LangID from the registry.

Is it OK if they're different?

Reply:
Yes. `-default-lang` is providing information about the user's default language. `-register-tip` is attempting to register for a specific language. The reason we pass the user's default language to the elevated process is in case of fallback.